.github/workflows/conventional_commits_title.yml

```diff
@@ -0,0 +1,26 @@
+# Validates PR title follows conventional commits
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  conventional_commit_title:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5
```

Comment:
You can pin this script to our github actions repo too like in here: https://github.com/chanzuckerberg/shared-infra/blob/main/.github/workflows/conventional_commits_title.yml#L14

Reply:
thank you so much! Yeah, I think that's a good recommendation. I appreciate it